libc/src/__support/FPUtil/riscv/FEnvImpl.h

```diff
@@ -12,7 +12,6 @@
 #include "src/__support/FPUtil/FPBits.h"
```

Comment:
Add `#include "hdr/fenv_macros.h"` here also. It seems redundant, but it will make the next step easier.